frontend/packages/console-plugin-sdk/src/typings/details.ts

```diff
@@ -0,0 +1,23 @@
+import { K8sKind } from '@console/internal/module/k8s';
+import { PageComponentProps } from '@console/internal/components/utils/horizontal-nav';
+import { Extension } from './extension';
+
+namespace ExtensionProperties {
+  export interface DetailViewTab {
+    model: K8sKind;
+    flag?: string;
```

Comment:
Just wait for https://github.com/openshift/console/pull/3383

Reply:
Can I help with review or testing to get that pull request merged?

Reply:
@alecmerdler by all means.
There's just one pending typing fix left on my side for it.